protected/humhub/modules/admin/views/group/edit.php

```diff
@@ -38,6 +38,21 @@
         ?>
     <?php endif; ?>
 
+    <strong><?= Yii::t('AdminModule.user', 'Users notification'); ?></strong>
+    <br>
+    <br>
+
+    <?= $form->field($group, 'notify_users')->checkbox(); ?>
+    <div class="help-block">
+    <?= Yii::t(
+        'AdminModule.user',
+        'Notify users about adding to the group'
```

Comment:
Notify users when adding them to this group

Reply:
Done!

Reply:
@MasterHans I've moved this to the attribute hints: https://github.com/humhub/humhub/pull/4643/files#diff-a48b33b5fa0409f1ae133b00086682c396c4f4ea470d30d47823c15231284e9dR99